about/process/issue-tracking.rst

```diff
@@ -1,105 +1,108 @@
 Issue-Tracking Guidelines
 =========================
 
-This document describes the standard process of dealing with new issues in UBports projects. Not to be confused with the :doc:`guide on writing a good bugreport </contribute/bugreporting>`.
+This document describes the standard process of dealing with new issues in UBports projects.
+(Not to be confused with the :doc:`guide on writing a good bugreport </contribute/bugreporting>`.)
 
 Where are bugs tracked?
 -----------------------
 
-Since our quality assurance depends heavily on the community, we try to
-track issues where the user would expect them, instead of separated by
-repository. This means, that issues of almost all components that are
-distributed as with the system-image are tracked in the
-`Ubuntu Touch tracker <https://github.com/ubports/ubuntu-touch>`__. An
-exception of this are click-apps that can be updated independently through
+Since quality assurance depends heavily community, issues are
+tracked where users expects them, instead of separated by repository.
+This means issues of almost all distributed components (as with the system-image)
+are tracked in the `Ubuntu Touch tracker <https://github.com/ubports/ubuntu-touch>`__.
+An exception to this are click-apps, which can be updated independently through
 the OpenStore.
 
-Most other repositories track issues locally. If you're unsure whether a
-repository uses its own tracker or not, consult the README.md file.
-Repositories that don't track issues locally have their bugtracker disabled.
+Most other repositories track issues locally. You will find out whether a
+repository uses its own tracker or not in its README.md file.
+Repositories that don't track issues locally have their bugtracker turned off.
 
 This page is mainly about the Ubuntu Touch tracker, but most principles apply
-to other projects as well.
+to other projects.
 
 .. note::
-     Practicality beats purity! Exceptions might apply and should be described in the projects README.md file.
+     Practical exceptions to purity might apply and must (no exceptions) be described in the project's README.md file.
```

Comment:
If you do fluff, you can do intrigue.

Reply:
i dont understand

Reply:
It is a bit of a joke, somewhat a conundrum. There is no exclamation mark thrown at the reader. It is fine.

Reply:
We're talking past each other. My original comment was that in my opinion the original wording was clearer. The use of the word purity was clear as it was introduced to contrast with practicality. In the new proposal, it seems that the word purity is just falling from the sky. The original phrasing was light hearted and intuitively clear. The new phrasing is light hearted but confusing. Also should it be 'may' not 'might'?. Lastly and most importantly, I disagree with the new content! That exceptions **must** be described. I think this is the opposite of the original statement about Practicality beats purity. And not only are we turning around the meaning, we are extra, extra emphasizing it with an additional "no exceptions" -- geez. chill. why the urgency? Even if I would agree that it's a good ideal to be zealously pure here - it's just unrealistic. In the real world it will never happen that all exceptions are diligently documented. And I don't think they should have to be! Because, you know what, Practicality **does** beat purity. So, I maintain: Please revert.